packages/config/src/loadConfigFromFile.ts

```diff
@@ -106,7 +106,10 @@ export async function loadConfigFromFile({
     }
 
     return {
-      config: required['default'] as PrismaConfig,
+      config: {
+        ...(required['default'] as PrismaConfig),
+        loadedFromFile: resolvedPath,
```

Comment:
ℹ️ Based on this property we can detect whether a `prisma.config.ts` file was used/found.